### PR TITLE
k3d cluster cpu limit

### PR DESCRIPTION
**Pull request by asafyehezkel**
- Add cpu limit flag
- Apply cpu limit on ked cluster config

### CODE REVIEW

File: pkg/server/flags.go [hunk 1]
```diff
@@ -61,5 +62,6 @@ func (flags *InstallFlags) SetFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&flags.ProxyUrl, "proxy-url", "", "Proxy URL to be used for tensorleap server")
 	cmd.Flags().BoolVar(&flags.FixK3dDns, "fix-dns", false, "Fix DNS issue with docker, in case you are having issue with internet connection in the container")
 	cmd.Flags().StringVarP(&flags.DataDir, "data-dir", "d", "", "Directory to store tensorleap data, by default using /var/lib/tensorleap/standalone or previous data directory")
+	cmd.Flags().StringVar(&flags.CpuLimit, "cpu-limit", "", "Limit the CPU resources for the k3d cluster (e.g. 2 for 2 cores)")
```
Comment (moshe-kabala) [inline pkg/server/flags.go:65]: consider to improve the description (k3d cluster - user not really understand what is k3d)

File: pkg/k3d/cluster.go [hunk 2]
```diff
@@ -82,9 +86,59 @@ func CreateCluster(ctx context.Context, manifest *manifest.InstallationManifest,
...
+		} else {
+			fmt.Printf("CPU limit applied to container %s\n", container)
+		}
+	}
+
+	return nil
+}
+
+func getK3dContainers() ([]string, error) {
+	cmd := exec.Command("docker", "ps", "--filter", "name=k3d", "--format", "{{.ID}}")
```
Comment (moshe-kabala) [inline pkg/k3d/cluster.go:120]: consider the case user has two k3d cluster

**Review by moshe-kabala — APPROVED** [review]
GREAT JOB
consider to keep the number of CPU in case of install (like we did for domain)

File: pkg/k3d/cluster.go (hunk 2 above)
Comment (asafyehezkel) [reply pkg/k3d/cluster.go:120]: Changed to `k3d-tensorleap`

File: pkg/server/flags.go (hunk 1 above)
Comment (asafyehezkel) [reply pkg/server/flags.go:65]: It is chartGPT description 😆  I thought the example made some order no? `(e.g. 2 for 2 cores)`

**Pull request merged**